test/tests/performance/pbs_cgroups_stress.py

```diff
@@ -191,10 +180,12 @@ def load_config(self, cfg):
         # Restart MoM to work around PP-993
         self.mom.restart()
 
-    @timeout(1800)
+    @timeout(600)
     def test_cgroups_epilogue_race(self):
         """
-        Test to verify cgroups hook does not read/write  needed files
+        Test to ensure a cgroups event does not read the cgroups file system
+        while another event is reading it. By submitting 1000 instant jobs,
```

Comment:
I think you mean an event does not read the cgroups file system while another event is writing to it.